test/foundry/TransferEth.t.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.22;
+
+/* solhint-disable func-name-mixedcase */
+
+import "./utils/TransferEthScenario.sol";
+
+/*
+ * Runs tests for more complex scenarios
+ */
+contract TransferEth is TransferEthScenario {
+    function setUp() public override {
+        super.setUp();
+        super.transferEth_setUp();
+    }
+
+    function test_transferEth() public {
+        uint256 accountInitialERC20Balance = _testERC20.balanceOf(address(_account));
+        uint256 accountInitialETHBalance = address(_account).balance;
+
+        //execute a swap intent
+        (uint256 erc20ReleaseAmount, uint256 transferAmount) = transferEth_run(false);
+
+        //verify end state
+        uint256 solverTokenBalance = _testERC20.balanceOf(_publicAddressSolver);
+        assertEq(solverTokenBalance, erc20ReleaseAmount, "The solver ended up with incorrect token balance");
+
+        uint256 userTokenBalance = _testERC20.balanceOf(address(_account));
+        uint256 expectedUserTokenBalance = accountInitialERC20Balance - erc20ReleaseAmount;
+        assertEq(userTokenBalance, expectedUserTokenBalance, "The user ended up with incorrect token balance");
+
+        uint256 recipientBalance = address(_recipientAddress).balance;
+        assertEq(recipientBalance, transferAmount, "The recipient didn't get the expected ETH");
```

Comment:
GitHub thinks the apostrophe here finishes the string that starts with the quotation mark. In the PR as well as the regular code viewer, it looks like everything else that is following the apostrophe is a string. This is not actually the case, so we can ignore this, add an escape for the apostrophe, or get rid of the contraction.